pkgs/tools/misc/pls/default.nix

```diff
@@ -2,14 +2,14 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "pls";
-  version = "4.0.3";
+  version = "5.0.0";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "dhruvkb";
-    repo = "pls";
+    repo = pname;
```

Comment:
I am not a fan of this as it makes it harder to copy source links together.

Reply:
No problem, I removed this change.